SConstruct

```diff
@@ -131,17 +131,20 @@ def set_defaults(env, daos_version):
     env.Append(CCFLAGS=['-DDAOS_VERSION=\\"' + daos_version + '\\"'])
     env.Append(CCFLAGS=['-DAPI_VERSION=\\"' + API_VERSION + '\\"'])
 
-def build_misc():
+def build_misc(build_prefix):
     """Build miscellaneous items"""
     # install the configuration files
-    SConscript('utils/config/SConscript')
+    SConscript('utils/config/SConscript', variant_dir="{}/utils/config".format(build_prefix),
```

Comment:
I think I would prefer f-strings or even os.path.join here, same for a lot of other code in this PR.

Reply:
What is the difference between f-strings and format?   I see a lot of people using the latter so I assumed it's preferred generally.

Also, is there any benefit to os.path.join outside of supporting Windows?

Reply:
f-strings are newer and easier to read, they didn't work on python-2 however.

The benefit in os.path.join again is that it's clear that what you're modifying is a path string so I think either would be acceptable in this case.